### PR TITLE
Fix/TR-3312/clean vulnerabilities

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/TR-3312

- Cleaned vulnerabilities
- Fixed readme
- Updated supportBrowsers

image

### CODE REVIEW

**Comment by bziondik** [comment]
@jsconan @btamas do you know where we use this npm package? :)

**Review by Karol-Stelmaczonek — APPROVED** [review]

**Review by jsconan — APPROVED** [review]
Good job, coming from 41 vulnerabilities down to 0.

I've added a comment, non-blocking, regarding the version of the browser list. It would tick a box in the upgrade plan listed here https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/1658847972/IE+11+-+Support+and+End+Of+Life?focusedCommentId=2015461422

**Comment by jsconan** [comment]
@bziondik this quick search should help you finding where is this package consumed: https://github.com/search?l=JSON&q=org%3Aoat-sa+browser-support-checker&type=Code

File: package.json [hunk 1]
```diff
@@ -33,16 +33,16 @@
   "devDependencies": {
     "@babel/core": "^7.13.15",
     "@babel/preset-env": "^7.13.15",
+    "@oat-sa/browserslist-config-tao": "^0.3.0",
```
[the remark this answers is not in the document]
Comment (bziondik) [reply package.json]: Updated and IE removed from [supportedBrowsers.js](https://github.com/oat-sa/browser-support-checker/pull/15/commits/e15a3a092d37c169f37e1f9e0d3aa90cef376286#diff-957b784ee64b828a362a83a81f2354d1ed4e4e36489e53c33b9dae010533a6c2) 👌😁

**Pull request merged**